docs/src/pages/components/tree-view/tree-view.md

```diff
@@ -62,3 +62,7 @@ const data = {
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#TreeView)
 
 The component follows the WAI-ARIA authoring practices.
+
+To have an accessible tree view:
```

Comment:
Does it make sense to separate this from the single bullet that follows? (Or to have a bullet list with one item?)

Reply:
This was done in anticipation of more items but we can make it a list when we have any :P